diff --git a/dist/rrweb.cjs b/dist/rrweb.cjs
index 7d09b21b99346971694a67ca06e3a9a1655232b9..48fd3a1cc9ec621a35fda0730393e24a8951dfd6 100644
--- a/dist/rrweb.cjs
+++ b/dist/rrweb.cjs
@@ -14737,6 +14737,10 @@ function createPlayerService(context, { getCastFn, applyEventsSynchronously, emi
         },
         live: {
           on: {
+            STOP_LIVE: {
+              target: "paused",
+              actions: ["pause"]
+            },
             ADD_EVENT: {
               target: "live",
               actions: ["addEvent"]
@@ -15863,6 +15867,9 @@ class Replayer {
   startLive(baselineTime) {
     this.service.send({ type: "TO_LIVE", payload: { baselineTime } });
   }
+  stopLive() {
+    this.service.send({ type: "STOP_LIVE" });
+  }
   addEvent(rawEvent) {
     const event = this.config.unpackFn ? this.config.unpackFn(rawEvent) : rawEvent;
     if (indicatesTouchDevice(event)) {
diff --git a/dist/rrweb.d.cts b/dist/rrweb.d.cts
index 92d2c5f26748fea6b13d384e815887802fefd013..4645f0cec23dad1f527f9c4490af1348d70923ce 100644
--- a/dist/rrweb.d.cts
+++ b/dist/rrweb.d.cts
@@ -178,6 +178,8 @@ declare type PlayerEvent = {
     };
 } | {
     type: 'PAUSE';
+} | { 
+    type: 'STOP_LIVE'
 } | {
     type: 'TO_LIVE';
     payload: {
@@ -295,6 +297,7 @@ export declare class Replayer {
     resume(timeOffset?: number): void;
     destroy(): void;
     startLive(baselineTime?: number): void;
+    stopLive(): void;
     addEvent(rawEvent: eventWithTime | string): void;
     enableInteract(): void;
     disableInteract(): void;
diff --git a/dist/rrweb.d.ts b/dist/rrweb.d.ts
index 92d2c5f26748fea6b13d384e815887802fefd013..b4cd06187812104a79a543a92977ccaa497ba4a2 100644
--- a/dist/rrweb.d.ts
+++ b/dist/rrweb.d.ts
@@ -178,6 +178,8 @@ declare type PlayerEvent = {
     };
 } | {
     type: 'PAUSE';
+} | {
+    type: 'STOP_LIVE';
 } | {
     type: 'TO_LIVE';
     payload: {
@@ -295,6 +297,7 @@ export declare class Replayer {
     resume(timeOffset?: number): void;
     destroy(): void;
     startLive(baselineTime?: number): void;
+    stopLive(): void;
     addEvent(rawEvent: eventWithTime | string): void;
     enableInteract(): void;
     disableInteract(): void;
diff --git a/dist/rrweb.js b/dist/rrweb.js
index 553c605c1a85481634140101148d705518a06619..bf92559455d35b122035db896597d6fd5c6a1248 100644
--- a/dist/rrweb.js
+++ b/dist/rrweb.js
@@ -14735,6 +14735,10 @@ function createPlayerService(context, { getCastFn, applyEventsSynchronously, emi
         },
         live: {
           on: {
+            STOP_LIVE: {
+              target: "paused",
+              actions: ["pause"]
+            },
             ADD_EVENT: {
               target: "live",
               actions: ["addEvent"]
@@ -15861,6 +15865,9 @@ class Replayer {
   startLive(baselineTime) {
     this.service.send({ type: "TO_LIVE", payload: { baselineTime } });
   }
+  stopLive() {
+    this.service.send({ type: "STOP_LIVE" });
+  }
   addEvent(rawEvent) {
     const event = this.config.unpackFn ? this.config.unpackFn(rawEvent) : rawEvent;
     if (indicatesTouchDevice(event)) {
