diff --git a/node_modules/rrweb/dist/rrweb.cjs b/node_modules/rrweb/dist/rrweb.cjs
index d455aa0..86e7de1 100644
--- a/node_modules/rrweb/dist/rrweb.cjs
+++ b/node_modules/rrweb/dist/rrweb.cjs
@@ -14576,6 +14576,10 @@ function createPlayerService(context, { getCastFn, applyEventsSynchronously, emi
         }, 
         live: {
           on: {
+            STOP_LIVE: {
+              target: "paused",
+              actions: ["pause"]
+            },
             ADD_EVENT: {
               target: "live",
               actions: ["addEvent"]
@@ -15702,6 +15706,9 @@ class Replayer {
   startLive(baselineTime) {
     this.service.send({ type: "TO_LIVE", payload: { baselineTime } });
   }
+  stopLive() {
+    this.service.send({ type: "STOP_LIVE" });
+  }
   addEvent(rawEvent) {
     const event = this.config.unpackFn ? this.config.unpackFn(rawEvent) : rawEvent;
     if (indicatesTouchDevice(event)) {
diff --git a/node_modules/rrweb/dist/rrweb.d.ts b/node_modules/rrweb/dist/rrweb.d.ts
index 95305d6..66540cb 100644
--- a/node_modules/rrweb/dist/rrweb.d.ts
+++ b/node_modules/rrweb/dist/rrweb.d.ts
@@ -182,6 +182,8 @@ declare type PlayerEvent = {
     };
 } | {
     type: 'PAUSE';
+} | {
+    type: 'STOP_LIVE';
 } | {
     type: 'TO_LIVE';
     payload: {
@@ -299,6 +301,7 @@ export declare class Replayer {
     resume(timeOffset?: number): void;
     destroy(): void;
     startLive(baselineTime?: number): void;
+    stopLive(): void;
     addEvent(rawEvent: eventWithTime | string): void;
     enableInteract(): void;
     disableInteract(): void;
diff --git a/node_modules/rrweb/dist/rrweb.js b/node_modules/rrweb/dist/rrweb.js
index 5d38597..13a9d10 100644
--- a/node_modules/rrweb/dist/rrweb.js
+++ b/node_modules/rrweb/dist/rrweb.js
@@ -14574,6 +14574,10 @@ function createPlayerService(context, { getCastFn, applyEventsSynchronously, emi
         },
         live: {
           on: {
+            STOP_LIVE: {
+              target: 'paused',
+              actions: ["pause"]
+            }, 
             ADD_EVENT: {
               target: "live",
               actions: ["addEvent"]
@@ -15700,6 +15704,9 @@ class Replayer {
   startLive(baselineTime) {
     this.service.send({ type: "TO_LIVE", payload: { baselineTime } });
   }
+  stopLive() {
+    this.service.send({ type: "STOP_LIVE" });
+  }
   addEvent(rawEvent) {
     const event = this.config.unpackFn ? this.config.unpackFn(rawEvent) : rawEvent;
     if (indicatesTouchDevice(event)) {
